app/workers/dormant_user_mailer_worker.rb

```diff
@@ -0,0 +1,12 @@
+class DormantUserMailerWorker
+  include Sidekiq::Worker
+
+  attr_reader :user
+
+  def perform(user_id)
+    @user = User.find(user_id)
```

Comment:
This should also short circuit and return straight away if we are on the staging env (non production env). If there are users on there they don't want to be getting emails that make no sense.

Reply:
Noting here that we have opted to not make it a scheduled job. Instead, we'll enqueue it a few times with the correct cohorts at the start, and we'll do that on production. Acceptable?